pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -394,6 +395,22 @@ func (c *Client) ValidateServerForOperator(operator *v1beta1.Operator) error {
 	return nil
 }
 
+func (c *Client) CreateNamespace(namespace, manifest string) error {
+
+	ns := &v1core.Namespace{}
+	if manifest != "" {
+		if err := yaml.Unmarshal([]byte(manifest), ns); err != nil {
+			return fmt.Errorf("unmarshalling namespace manifest file: %w", err)
+		}
+	}
+	ns.TypeMeta.Kind = "Namespace"
+	ns.Name = namespace
+	ns.Annotations["created-by"] = "kudo-cli"
```

Comment:
What is this annotation needed for? And if it's actually needed, let's put it into [labels.go](https://github.com/kudobuilder/kudo/blob/master/pkg/util/kudo/labels.go) with other annotations and prefix it with `kudo.dev/???`